modules/EvManager/manifest.yaml

```diff
@@ -76,6 +76,14 @@ config:
     description: Support three phase
     type: boolean
     default: true
+  departure_time:
```

Comment:
It is not enough to add the two parameters in the manifest.yaml. You should update the `EvManager` module with `ev-cli`. In this case, however, it is enough to add both parameters to the config struct: https://github.com/EVerest/everest-core/blob/main/modules/EvManager/EvManager.hpp#L28
Then the `EvManager` module has access to both static parameters.